components/constellation/constellation.rs

```diff
@@ -5466,4 +5469,39 @@ where
             error!("Got a media session action but no active media session is registered");
         }
     }
+
+    fn handle_gamepad_msg(&mut self, event: GamepadEvent) {
```

Comment:
Let's add doc comment to the method even it's evident in this case. People who read the code may want to know who is handling this message.